src/modules/kdenlive/CMakeLists.txt

```diff
@@ -2,9 +2,16 @@ if(NOT MOD_KDENLIVE)
     return()
 endif()
 
+find_package(Qt5 COMPONENTS Core Xml)
```

Comment:
Can you explain why the changes in this file are required for the typewriter module? I do not see where the typewriter code uses Qt.

Reply:
The `filter_typewriter.cpp` does almost not (except lines 228-229), but there is whole c++ class in `typewriter.cpp` which renders the effects frames and it uses Qt5::Xml for parsing xml from kdenlive to extract text content.